packages/app/src/DOMPlayer.tsx

```diff
@@ -2,6 +2,7 @@ import { useCallback, useEffect, useRef, useState } from 'react';
 import { useHotkeys } from 'react-hotkeys-hook';
 import { Replayer } from 'rrweb';
 import { throttle } from 'lodash';
+import cn from 'classnames';
```

Comment:
```suggestion
import cx from 'classnames';
```

Just a nit - we import `classnames` as `cx` (tl;dr due to my time at fb...) - would be awesome to have this import aligned with that as well.

Reply:
Thank you for taking a look! I just pushed the update, I will keep it in mind going forward 😃